nix/tests/vm.nix

```diff
@@ -14,4 +14,4 @@ let
     all = pkgs.callPackage ./vm-tests/all.nix { inherit makeTest plutus-playground web-ghc docs vmCompileTests; };
   };
 in
-if isDarwin then { } else tests
```

Comment:
As discussed, Plutus-playground is dying so these tests are obsolete.
In the next PR I'll actually delete the playground and related nix files